api/src/test/java/io/kafbat/ui/service/quota/ClientQuotaServiceTest.java

```diff
@@ -68,12 +73,16 @@ void createUpdateDelete(String user, String clientId, String ip) {
         .assertNext(status -> assertThat(status.value()).isEqualTo(204))
         .verifyComplete();
 
-    assertThat(quotaRecordExists(new ClientQuotaRecord(user, clientId, ip, Map.of("producer_byte_rate", 22222.0))))
-        .isFalse();
+    awaitAndVerify((l) ->
+        l.doesNotContain(new ClientQuotaRecord(user, clientId, ip, Map.of("producer_byte_rate", 22222.0)))
+    );
   }
 
-  private boolean quotaRecordExists(ClientQuotaRecord rec) {
-    return Objects.requireNonNull(quotaService.getAll(cluster).collectList().block()).contains(rec);
+  private void awaitAndVerify(Consumer<ListAssert<ClientQuotaRecord>> verifier) {
+    await()
```

Comment:
do we need await here?

Reply:
Yes. because nature of Kafka KRaft protocol